<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomRequestDialogUI</class>
 <widget class="QDialog" name="CustomRequestDialogUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>857</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>742</width>
     <height>827</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <widget class="QPlainTextEdit" name="headers">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="send">
      <property name="text">
       <string>SEND</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Method: </string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Headers:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPlainTextEdit" name="body"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="newHeaderName">
      <property name="text">
       <string>Header Name</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QPlainTextEdit" name="log">
      <property name="minimumSize">
       <size>
        <width>640</width>
        <height>480</height>
       </size>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="method">
      <item>
       <property name="text">
        <string>HEAD</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>GET</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PUT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>POST</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PROPFIND</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PROPPATCH</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MKCOL</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DELETE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MOVE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>TRACE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>OPTIONS</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>URL: </string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="addHeader">
      <property name="text">
       <string>Add Header</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Body: </string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="url">
      <property name="text">
       <string>https://webdav.yandex.ru/</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="clear">
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="newHeaderValue">
      <property name="text">
       <string>Header Value</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>LOG:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
